admin-client/src/pages/domain/Show.svelte

```diff
@@ -0,0 +1,82 @@
+<script>
+  import { router } from '../../stores';
+  import { fetchDomain, fetchDomainDnsResult, provisionDomain } from '../../lib/api';
+  import { formatDateTime } from '../../helpers/formatter';
+  import { siteName } from '../../lib/utils';
+  import {
+    Await,
+    GridContainer,
+    PageTitle,
+    LabeledItem,
+  } from '../../components';
+  import { domainBranch, stateColor } from './domain';
+  import DnsTable from './DnsTable.svelte';
+
+  $: id = $router.params.id;
+  $: domainPromise = fetchDomain(id);
+  $: dnsResultsPromise = fetchDomainDnsResult(id);
+
+  function refreshDnsResults() {
+    dnsResultsPromise = fetchDomainDnsResult(id);
+  }
+
+  function provision() {
+    domainPromise = provisionDomain(id);
+  }
+</script>
+
+<GridContainer>
+  <PageTitle>
+    Domain {id}
+  </PageTitle>
+  <Await on={domainPromise} let:response={{ dnsRecords, domain }}>
+    <h3>
+      <div class="display-flex flex-justify flex-align-center font-sans-lg">
+        <span>{domain.names}</span>
+        <span class="usa-tag radius-pill padding-y-1 {stateColor(domain.state)}">{domain.state}</span>
+      </div>
+    </h3>
+    <div class="grid-row">
+      <div class="tablet:grid-col-fill padding-bottom-1">
+        <LabeledItem label="id" value={domain.id} />
+        <LabeledItem label="site" value={siteName(domain.Site)} />
+        <LabeledItem label="context" value={domain.context} />
+        <LabeledItem label="branch" value={domainBranch(domain)} />
+        {#if domain.state !== 'pending'}
```

Comment:
`pending` is the initial state and `provisioning` is while it is being provisioned. These are not shown in the `pending` state because they don't exist yet, they are set when attempting to provision the domain. I think the question here is does this approach make sense or should we set them when initially creating the domain.